src/components/authors/authors.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Alata&display=swap');
+
+.container {
+  color: #fff;
+  font-family: 'Alata', sans-serif;
+  font-size: 3em;
+  height: 406px;
+  line-height: 1.4em;
+  margin: 0 auto;
+  text-align: center;
+  text-shadow: 10px 20px 10px rgba(70, 6, 15, 0.62);
+  width: 866px;
+};
+
+a {
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
This is not how BEM works. You are not supposed to style tags unless you have a very good reason.